frameworks/helloworld/tests/test_sanity.py

```diff
@@ -181,13 +183,18 @@ def test_state_properties_get():
     foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
 
     jsonobj = sdk_cmd.svc_cli(config.PACKAGE_NAME, foldered_name, 'state properties', json=True)
-    assert len(jsonobj) == 5
-    # alphabetical ordering:
-    assert jsonobj[0] == "hello-0-server:task-status"
-    assert jsonobj[1] == "hello-1-server:task-status"
-    assert jsonobj[2] == "last-completed-update-type"
-    assert jsonobj[3] == "world-0-server:task-status"
-    assert jsonobj[4] == "world-1-server:task-status"
+    # should be in alphabetical order:
+    expected = [
+        "hello-0-server:task-status",
+        "hello-1-server:task-status",
+        "last-completed-update-type",
+        "world-0-server:task-status",
+        "world-1-server:task-status"]
+    # the properties list may also have a 'suppressed' bit, which would have been left behind by the
+    # prior version when upgrades were being tested during suite setup
+    expected_with_suppressed = list(expected)
```

Comment:
Would it not be better to remove "suppressed" from the result set and test for equality then?

Reply:
That would work too. With this method we at least verify that the suppressed bit, if present, is at the expected index in the returned list.